Nos/Views/SideMenuContent.swift

```diff
@@ -19,18 +19,15 @@ struct SideMenuContent: View {
     let closeMenu: () -> Void
     
     var body: some View {
-        ZStack {
-            Color(UIColor(red: 255 / 255.0, green: 255 / 255.0, blue: 255 / 255.0, alpha: 1))
-            
-            VStack(alignment: .leading, spacing: 0, content: {
+        NavigationStack(path: $router.sideMenuPath) {
+            VStack(alignment: .leading, spacing: 0) {
+                Spacer()
                 HStack {
                     Button {
                         do {
                             guard let keyPair = KeyPair.loadFromKeychain() else { return }
                             let author = try Author.findOrCreate(by: keyPair.publicKeyHex, context: viewContext)
-                            router.path.append(author)
-                            router.navigationTitle = Localized.profile.string
-                            closeMenu()
+                            router.sideMenuPath.append(author)
```

Comment:
So we could use router.push here instead, right?

Reply:
yeah, I didn't end up using push in all the places I could.